frontend/packages/knative-plugin/src/components/functions/index.ts

```diff
@@ -0,0 +1,10 @@
+export { CreateActionDropdown } from './CreateActionDropdown';
```

Comment:
this file is referenced in https://github.com/openshift/console/blob/3f55ee6600809875a6b31b95f7f82d87f9ddea32/frontend/packages/knative-plugin/package.json#L35 but the corresponding `index.ts` file does not exist